Source/CombatExtended/CombatExtended/Comps/HediffCompProperties_InfecterCE.cs

```diff
@@ -8,13 +8,8 @@
 
 namespace CombatExtended
 {
-    public class HediffCompProperties_InfecterCE : HediffCompProperties
+    public class HediffCompProperties_InfecterCE : HediffCompProperties_Infecter
```

Comment:
What's the rationale behind this change?

Reply:
HediffComp_InfecterCE was based on it and I thought it was just a copy from an old Rimworld version. Also this allowed me to fix other things for now.